Fabric.Identity.API/Models/IdentityResource.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+
+namespace Fabric.Identity.API.Models
+{
+    public class IdentityResource
```

Comment:
Do we need to review the IS4 model defaults here? I think we should use the same defaults.